testing/test_aux.sh

```diff
@@ -191,4 +195,65 @@ head inst/run-port-*/finish/nmap*/*
 
 tcpdump -en -r inst/run-port-01/scans/test_nmap.pcap icmp or arp
 
+
+function monitor_interface {
```

Comment:
THis doesn't seem to be "monitor_inteface" -- it's really "monitor_log"?  The interface part is only incidental b/c of how it's called, not the function itself.